CSharp_Course_Project/Program.cs

```diff
@@ -6,7 +6,25 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Watta fuck??? Doesn't work");
+            squareCircle();
+            //Console.WriteLine(squareCircle2(4)); 
+        }
+
+        public static void squareCircle()
+        {
+            const double pi = Math.PI;
+            Console.Write("Enter a radius of a circle: ");
+            string radius = Console.ReadLine();
+            double r = Convert.ToInt32(radius);
```

Comment:
Please pay attention to the method you're using for parsing.
If you parse double, you need to store it in double.